test/tests/contacts/ContactUtilsTest.ts

```diff
@@ -177,4 +187,55 @@ o.spec("ContactUtilsTest", function () {
 		o(formatBirthdayNumeric(leapYearBirthday)).equals("29/02/2016")
 		o(formatBirthdayNumeric(chromeBugBirthday)).equals("15/08/1911")
 	})
+
+	o("extractStructuredEmailAddress", function () {
+		let contact = createTestEntity(ContactTypeRef)
+
+		contact.mailAddresses.push(createTestEntity(ContactMailAddressTypeRef))
+		contact.mailAddresses.push(createTestEntity(ContactMailAddressTypeRef))
+
+		o(contact.mailAddresses.length).equals(2)
```

Comment:
```suggestion

```

Correct me if I am wrong but `push()` is the function from the Array object in the standard library. Asserting whether `push()` actually inserted elements is just testing the standard library implementation which is not our responsibility.

 In addition, this is also an implementation detail of which tests should not check. Tests should only care whether the unit (in this case  `extractStructuredMailAddresses()`) fits it specifications. I.e. the function returns what it should.